features/journalist_can_edit_and_delete_articles.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Journalist can edit and delete articles
+
+    As a Journalist,
+    In order to change articles I have already written
+    I would like to be able to edit and delete articles
+    
+    
+    Background: 
+        Given the following categories exist
+            | name                  | 
+            | Breaking News         | 
+            | Politics              | 
+            | Sport                 | 
+            | Lifestyle             | 
+            | Health                | 
+
+        Given the following articles exist
```

Comment:
Refactor this whole feature file. Modify the background

Reply:
done

Reply:
The articles in the background need to associated with a journalist.

Reply:
done